recipes/svmbir/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "svmbir" %}
+{% set version = "0.2.7" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 45137732c9a345b570a03c205c37080d23def594245b62b5bc3b18b0ba5f3f6d
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - numpy =1.21
+  host:
+    - conda-build
+    - llvm-openmp  # [osx]
+    - cython
+    - pip
+    - python
+    - numpy =1.21
+  run:
+    - _openmp_mutex  # [linux]
```

Comment:
```suggestion
```

Reply:
Based on the build failure, it appears that this one was necessary.

Reply:
No, then you might need libgomp in host

Reply:
That seems to work.